src/e2e-test/resources/pluginParameters.properties

```diff
@@ -128,6 +128,11 @@ gcsReadRecursivePathSchema=[{"key":"id","value":"int"},{"key":"name","value":"st
   {"key":"yearofbirth","value":"int"},{"key":"tablename","value":"string"},{"key":"offset","value":"string"}]
 gcsParquetFile=testdata/GCS_PARQUET_TEST.parquet
 gcsAvroFile=testdata/GCS_AVRO_TEST.avro
+gcsAvroAllTypeData=[{"key":"id","value":"long"},{"key":"array","value":"string"},\
```

Comment:
Add schema text at the end of parameter.

Reply:
Done